From 385588ec1456d1b1f639699c6811c5d8e8405c91 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Mon, 27 Jun 2022 20:16:44 +0200
Subject: IWYU: vector used in gl_display.h requires include.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix build bug with libstdc++:
    ../../ui/gl/gl_display.h:80:5: error: ‘std::vector’ has not been declared
       80 |     std::vector<DisplayType>* init_displays);
          |     ^~~

Bug: 957519
Change-Id: Ib5cf8768f030d5ced945512c05f5eb6e4487898b
---
 ui/gl/gl_display.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/gl/gl_display.h b/ui/gl/gl_display.h
index a55edd5b223c2..46030cf4cb004 100644
--- a/ui/gl/gl_display.h
+++ b/ui/gl/gl_display.h
@@ -8,6 +8,7 @@
 #include <stdint.h>
 
 #include <memory>
+#include <vector>
 
 #include "ui/gl/gl_export.h"
 
-- 
2.34.1

